### PR TITLE
*：update replicas without restart

### LINKED ISSUE #221: [feature] Support change replicas without restart cluster.
**Is your feature request related to a problem? Please describe.**

Now use the `postStart` script to add nodes to the xenon cluster. This method will cause the cluster to be restarted when the number of replicas is changed.

**Describe the solution you'd like**

Add relevant logic to the operator instead of script.

step:

1. Set the cluster status to `Scale In` / `Scale out` when Replicas changes.

2. Wait for all Pods ready to make RAFT's master-slave status (Previous practice is starting at the Xenon container of each Pod)

3. If all Pod are new (init or closed), set the Pod of the serial number 0 to the leader directly

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by runkecheng**
### What type of PR is this?

/bug
/enhancement

### Which issue(s) this PR fixes?

Fixes #221 

### What this PR does?

Summary: Before the state is set to ready, check raft status and synchronize raft according to the spec.replicas.

### CODE REVIEW

File: internal/xenon_request.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"net/http"
+
+	"github.com/radondb/radondb-mysql-kubernetes/utils"
+)
+
+type xenonRequest struct {
```
Comment (runkecheng) [inline internal/xenon_request.go:28]: This part is overwriting.

File: utils/common.go [hunk 2]
```diff
@@ -111,6 +114,18 @@ func StringDiffIn(actual, desired []string) []string {
 	return diff
 }
 
+func StringDiffAdd(actual, desired []string) []string {
```
Comment (andyli029) [inline utils/common.go:117]: Merge StringDiffRemove into one function.

**Comment by acekingke** [comment]
LGTM

**Review by andyli029 — COMMENTED** [review]
Relicas-> Replicas

**Review by andyli029 — COMMENTED** [review]
LGTM.

**Pull request merged**